warehouse/query-core/src/main/java/datawave/query/iterator/logic/AndIterator.java

```diff
@@ -194,13 +276,15 @@ public T move(T minimum) {
         children.addAll(includes);
         children.addAll(excludes);
         
+        // TODO add deferred?
```

Comment:
These probably need to be added. The only references I could find were in QueryIterator.debugBooleanLogicIterator(). What I didn't want to do was cause the contextRequiredIterators to be initialized when not needed for evaluation. In this case it doesn't mater so I will add it. 

The leaves() method however does not have them and should not because the contextInclude/contextExclude iterators will be initialized and moved on demand.